autokeras/hypermodel/preprocessor.py

```diff
@@ -205,12 +184,24 @@ def transform(self, x):
     def output_types(self):
         return tf.int64,
 
+    @property
     def output_shape(self):
         return self.max_len or self._max_len,
 
+    def get_weights(self):
```

Comment:
These are not "weights" (numerical arrays), so prefer using a different name, e.g. save/load or get_state / set_state